cmd/antrea-agent/agent.go

```diff
@@ -439,41 +459,15 @@ func run(o *Options) error {
 		go ofClient.StartPacketInHandler(packetInReasons, stopCh)
 	}
 
-	// Initialize flow exporter to start go routines to poll conntrack flows and export IPFIX flow records
-	var conntrackPriorityQueue *priorityqueue.ExpirePriorityQueue
+	// Start go routines to periodically delete stale deny conections, poll conntrack
+	// flows and export IPFIX flow records.
 	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
-		v4Enabled := config.IsIPv4Enabled(nodeConfig, networkConfig.TrafficEncapMode)
-		v6Enabled := config.IsIPv6Enabled(nodeConfig, networkConfig.TrafficEncapMode)
-		isNetworkPolicyOnly := networkConfig.TrafficEncapMode.IsNetworkPolicyOnly()
+		denyConnStore := flowExporter.GetDenyConnStore()
+		go denyConnStore.RunPeriodicDeletion(stopCh)
 
-		conntrackPriorityQueue = priorityqueue.NewExpirePriorityQueue(o.activeFlowTimeout, o.idleFlowTimeout)
-		conntrackConnStore := connections.NewConntrackConnectionStore(
-			connections.InitializeConnTrackDumper(nodeConfig, serviceCIDRNet, serviceCIDRNetv6, ovsDatapathType, features.DefaultFeatureGate.Enabled(features.AntreaProxy)),
-			ifaceStore,
-			v4Enabled,
-			v6Enabled,
-			proxier,
-			networkPolicyController,
-			o.pollInterval,
-			conntrackPriorityQueue,
-			o.staleConnectionTimeout)
+		conntrackConnStore := flowExporter.GetConntrackConnStore()
 		go conntrackConnStore.Run(stopCh)
```

Comment:
could the `denyConnStore` and `conntrackConnStore` `Run` goroutines be started as part of `flowExporter.Run`?

Reply:
done